### PR TITLE
[Core]fix format str and log info

### PR DESCRIPTION
**Pull request by liuchang0812**
stageId's type is Int.

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala [hunk 1]
```diff
@@ -200,7 +200,7 @@ private[spark] class TaskSchedulerImpl(
         val execId = taskIdToExecutorId(tid)
         backend.killTask(tid, execId, interruptThread)
       }
-      tsm.abort("Stage %s cancelled".format(stageId))
+      tsm.abort("Stage %d was cancelled".format(stageId))
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala]: Better still, change this to use string interpolation. But, printing an int as an int value, and print its toString as a string, gives the same result. This doesn't do anything so I don't think I'd bother with this.

Comment (Liuchang0812) [inline core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala]: Yes, they have same result, but i think we should standard this. I would like change all of this to string interpolation if necessary.
thanks for your comment :)

Comment (srowen) [inline core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala]: Although I'd like to standardize too, I think it's not worth it as I've come to appreciate how annoying merge conflicts are. This just doesn't buy anything.

Comment (Liuchang0812) [inline core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala]: understand :smile: . close it now?

Comment (srowen) [inline core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala]: IMHO yes let's not bother with this sort of thing. It's a tiny change but doesn't improve anything and actually makes things a tiny bit more inconsistent.

**Pull request closed without merging**